sql/Pg-database.sql

```diff
@@ -4897,8 +4893,6 @@ select c.id, c.accno, coalesce(at.description, c.description),
 group by c.id, c.accno, coalesce(at.description, c.description), c.category,
          c.heading, c.gifi_accno, c.contra, c.tax;
 
-GRANT SELECT ON chart TO public;
-
 COMMENT ON VIEW chart IS $$Compatibility chart for 1.2 and earlier.$$;
```

Comment:
Maybe another PR might be to remove the chart view in 1.7

Reply:
Yea. I've thought of that. However, that would break the "incremental upgrades" in `sql/changes/`, because there's [drop_chart.sql](https://github.com/ledgersmb/LedgerSMB/blob/master/sql/changes/1.6/drop_chart.sql) upgrade script which would fail to apply.

The best way forward is probably by carefully designing a way to integrate the incremental scripts (automatically) into the schema file and making sure the upgrade scripts won't be applied anymore when they're "merged". It's something I've been thinking about, but not had the opportunity yet to write code for. (But, yes, this confused me before, so we seem to be at a point where the incrementals start to be confusing...)